eng/Versions.props

```diff
@@ -11,17 +11,17 @@
     <UsingToolNetFrameworkReferenceAssemblies>false</UsingToolNetFrameworkReferenceAssemblies>
     <!-- Other libs -->
     <AzureStorageBlobsVersion>12.6.0</AzureStorageBlobsVersion>
-    <MicrosoftAspNetCoreVersion>2.1.7</MicrosoftAspNetCoreVersion>
     <MicrosoftAspNetCoreAuthenticationNegotiateVersion>3.1.10</MicrosoftAspNetCoreAuthenticationNegotiateVersion>
-    <MicrosoftAspNetCoreHttpsPolicyVersion>2.1.1</MicrosoftAspNetCoreHttpsPolicyVersion>
+    <MicrosoftAspNetCoreHttpVersion>2.1.22</MicrosoftAspNetCoreHttpVersion>
     <MicrosoftAspNetCoreMvcVersion>2.1.3</MicrosoftAspNetCoreMvcVersion>
-    <MicrosoftAspNetCoreResponseCompressionVersion>2.1.1</MicrosoftAspNetCoreResponseCompressionVersion>
+    <MicrosoftAspNetCoreServerKestrelCoreVersion>2.1.7</MicrosoftAspNetCoreServerKestrelCoreVersion>
     <MicrosoftBclHashCodeVersion>1.1.0</MicrosoftBclHashCodeVersion>
     <MicrosoftExtensionsConfigurationKeyPerFileVersion>5.0.2</MicrosoftExtensionsConfigurationKeyPerFileVersion>
```

Comment:
Perhaps we should do ranges for all our packages? For example, 5.0.3 was recently released, and I'm sure 3.1.* is getting periodic security updates as well.

Reply:
From a packaging perspective, if you just specify an exact version, then the package dependency will be ">= {version}". For example, the Microsoft.Bcl.HasCode package dependency will be ">= 1.1.0". So all of the package references in the RestServer assembly had ">= {version}". This change is only constraining the ASP.NET Core 2.1 versions.

Other packages that are used only by the dotnet-monitor tool do not need version ranges because the tool itself is not a library so no one can reference it.